geotrek/api/v2/serializers.py

```diff
@@ -1056,6 +1060,7 @@ class CourseSerializer(PDFSerializerMixin, DynamicFieldsMixin, serializers.Model
         parents = serializers.ReadOnlyField(source='parents_id')
         accessibility = serializers.SerializerMethodField()
         attachments = AttachmentSerializer(many=True, source='sorted_attachments')
+        equipment = serializers.SerializerMethodField()
```

Comment:
I don't understand here why you delete equipment in fields but add attribute

Reply:
This is not for the same model it is for outdoor course (we should translate it)